config/application.rb

```diff
@@ -58,6 +58,19 @@ class Application < Rails::Application
     config.eager_load_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('app', 'validators')
 
+    # :nocov:
+    if Rails.env.production?
```

Comment:
I believe this needs to be gated behind a check if its actually running in public cloud mode, otherwise it will interfere with regular customers setups

Reply:
Added